include/odp/api/spec/crypto.h

```diff
@@ -90,7 +90,10 @@ typedef enum {
 	ODP_DEPRECATE(ODP_CIPHER_ALG_AES128_CBC),
 
 	/** @deprecated  Use ODP_CIPHER_ALG_AES_GCM instead */
-	ODP_DEPRECATE(ODP_CIPHER_ALG_AES128_GCM)
+	ODP_DEPRECATE(ODP_CIPHER_ALG_AES128_GCM),
+
+	/** AES with counter mode */
+	ODP_CIPHER_ALG_AES_CTR,
```

Comment:
I'd move this after AES_CBC or AES_GCM. It's easier to see what's deprecated when those are on the bottom.

Reply:
Doing so would break binary compatibility. Is it ok?

Reply:
Yes, since ABI compatibility is promised to be maintained within an API version, but not between API versions.